### PR TITLE
docs(content): update type to variant

### PR DESCRIPTION
**Pull request by aagonzales**
We've decided to go with the term "variant" instead of "type" to describe that different kinds of variations within a component. Updated all instances or "type" to read "variant" instead.

### CODE REVIEW

File: src/pages/components/breadcrumb/usage.mdx [hunk 1]
```diff
@@ -50,9 +50,9 @@ If you are taking users through a multistep process use a
 
 ### Types
 
-Carbon supports two types of breadcrumbs. Both types are styled the same, but
+Carbon supports two types of breadcrumbs. Both tyoes are styled the same, but
```
Comment (janchild) [inline src/pages/components/breadcrumb/usage.mdx:53]:
```suggestion
Carbon supports two types of breadcrumbs. Both types are styled the same, but
```

File: src/pages/components/breadcrumb/usage.mdx [hunk 2]
```diff
@@ -50,9 +50,9 @@ If you are taking users through a multistep process use a
...
 the methods for populating the breadcrumb trail are different. The breadcrumb
-type used should be consistent across a product.
+variant used should be consistent across a product.
```
Comment (janchild) [inline src/pages/components/breadcrumb/usage.mdx:55]:
```suggestion
type used should be consistent across a product.
```

**Review by janchild — APPROVED** [review]
Awesome, @aagonzales! Just two small fixes but then it's good to go.

**Review by joshblack — APPROVED** [review]
Looks great!

**Pull request merged**